src/components/backup/RestoreCloudStep.js

```diff
@@ -198,7 +198,16 @@ export default function RestoreCloudStep({
           const p1 = dispatch(walletsSetSelected(firstWallet));
           const p2 = dispatch(addressSetSelected(firstAddress));
           await Promise.all([p1, p2]);
-          await initializeWallet(null, null, null, false, false, null, true);
+          await initializeWallet(
+            null,
+            null,
+            false,
+            false,
+            null,
+            null,
+            null,
+            true
+          );
```

Comment:
I made this change because i made changes to the method signature [initializeWallet](https://github.com/rainbow-me/rainbow/pull/3464/files/dc48686ba1b3b329eacad3400cd1a2d5c596de58#diff-12d8f761ac899556e7c0f7538b03816a6e871c1105b73debf28c1ba76f63ac97)